artifactory-tasks-utils/utils.js

```diff
@@ -204,9 +204,14 @@ function configureCliServer(artifactory, serverId, cliPath, buildDir) {
 function deleteCliServers(cliPath, buildDir, serverIdArray) {
     let deleteServerIDCommand;
     for (let i = 0, len = serverIdArray.length; i < len; i++) {
-        deleteServerIDCommand = cliJoin(cliPath, cliConfigCommand, 'delete', quote(serverIdArray[i]), '--interactive=false');
-        // This operation throws an exception in case of failure.
-        executeCliCommand(deleteServerIDCommand, buildDir);
+        try{
```

Comment:
Please run `npm run format` before each commit
```suggestion
        try {
```